crates/typst/src/math/accent.rs

```diff
@@ -114,7 +114,12 @@ impl LayoutMath for Packed<AccentElem> {
     #[typst_macros::time(name = "math.accent", span = self.span())]
     fn layout_math(&self, ctx: &mut MathContext, styles: StyleChain) -> SourceResult<()> {
         let cramped = style_cramped();
-        let base = ctx.layout_into_fragment(self.base(), styles.chain(&cramped))?;
+        let mut base = ctx.layout_into_fragment(self.base(), styles.chain(&cramped))?;
+
+        // Try replace a glyph with its dotless variant.
+        if let MathFragment::Glyph(ref mut glyph) = &mut base {
+            glyph.make_dotless_form(ctx)
```

Comment:
(Moving part of the old conversation here.) Are there any situations where you'd want an accent above i or j _with_ the dot still? If this is basically never, it'd be much nicer to automatically replace them with their dotless variants.

Reply:
Some orthographies keep the dots of the i and j when accented, but I don’t know if there are equivalent math notations. There is at least one question on tex.stackexchange where this was desired https://tex.stackexchange.com/questions/181384/diacritics-above-dotted-i-i-with-tittle (though since you need to explicitly use `\imath` and `\jmath` to drop the dot in LaTeX, it is unlikely to have many questions asking about how to keep the dots).